Adafruit_ICM20649.cpp

```diff
@@ -204,33 +202,15 @@ bool Adafruit_ICM20649::_init(int32_t sensor_id) {
   sleep.write(false);    // take out of default sleep state
   clock_source.write(1); // AUTO SELECT BEST CLOCK
 
-  // set ACCEL
-  _setBank(2);
-  Adafruit_BusIO_Register accel_config_1 = Adafruit_BusIO_Register(
-      i2c_dev, spi_dev, ADDRBIT8_HIGH_TOREAD, ICM20649_ACCEL_CONFIG_1);
+  setGyroRange(ICM20649_GYRO_RANGE_1000_DPS);
```

Comment:
set default ranges to lowest ranges to start, ~250 dps, 2g

Reply:
Done, though the lowest for the ICM20649 are 500dps and 4g